docs/includes/query-builder/QueryBuilderTest.php

```diff
@@ -374,6 +374,18 @@ public function testWhereRegex(): void
         $this->assertInstanceOf(\Illuminate\Support\Collection::class, $result);
     }
 
+    public function testWhereLike(): void
+    {
+        // begin query whereLike
+        $result = DB::connection('mongodb')
+            ->table('movies')
+            ->whereLike('title', 'Start%', true)
```

Comment:
Note: this is causing the test to fail because the feature (#3108 ) is not merged yet

Reply:
https://github.com/mongodb/laravel-mongodb/pull/3108 is merged.